docs/specs/openapi.yaml

```
@@ -474,227 +506,111 @@ paths:
                     
                     Each file is a separate part.
                     
-                    For the file part, "Content-Type" is "application/octet-stream".
-
-                    "Content-Disposition" is "form-data; name="files"; filename=foo".
-          application/json:  # actin: make-dirs, remove
-            schema:
-              oneOf:  # Use oneOf since only one action can be performed at a time.
-                - $ref: "#/components/schemas/PostFilesMakeDirsRequest"
-                - $ref: "#/components/schemas/PostFilesRemovePathsRequest"
-            description: JSON payload for "make-dirs" or "remove" actions.
-      responses:
-        "200":
-          description: The result in the response is a JSON array of the file result object containing path and (optional) errors.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/PostFilesResponse"
-              example:
-                {"type":"sync","status-code":200,"status":"OK","result":[{"path":"/home/ubuntu/foo"}]}
-  /v1/exec:
-    post:
-      summary: Execute a command
-      tags:
-        - files
-      description: Start a command with the given options and return a value representing the process.
-      requestBody:
-        required: true
-        content:
-          application/json:
+                    For the file part, "Content-Type" is "application/octet-stream".
+
+                    "Content-Disposition" is "form-data; name="files"; filename=foo".
+          application/json:  # actin: make-dirs, remove
             schema:
-              $ref: "#/components/schemas/PostExecRequest"
-            example:
-              command: ["/bin/ls", "-l"]
+              oneOf:  # Use oneOf since only one action can be performed at a time.
+                - $ref: "#/components/schemas/PostFilesMakeDirsRequest"
+                - $ref: "#/components/schemas/PostFilesRemovePathsRequest"
+            description: JSON payload for "make-dirs" or "remove" actions.
       responses:
-        "202":
-          description: Command execution initiated.
+        "200":
+          description: Post files successfully. The result in the response is a JSON array of the file result object containing path and (optional) errors.
```

Since this endpoint can do several different operations, I think a more generic description would be better - saying "post files successfully" suggests to me that a file has been sent, which won't necessarily be the case. How about saying this (matching another endpoint):
```suggestion
          description: Successful operation. The result in the response is a JSON array of the file result object containing path and (optional) errors.
```